dev/io.openliberty.jaxws.config_fat/test-applications/simpleTestService/src/io/openliberty/jaxws/fat/stubclient/client/SimpleEchoService.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.jaxws.fat.stubclient.client;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.logging.Logger;
+
+import javax.xml.namespace.QName;
+import javax.xml.ws.Service;
+import javax.xml.ws.WebEndpoint;
+import javax.xml.ws.WebServiceClient;
+import javax.xml.ws.WebServiceFeature;
+
+@WebServiceClient(name = "SimpleEchoService", targetNamespace = "http://stubclient.fat.jaxws.openliberty.io/", wsdlLocation = "http://localhost:8010/simpleTestService/SimpleEchoService?wsdl")
```

Comment:
Will the hardcoded port (8010) work all the time? Is the provider guaranteed to start with this port?

Reply:
It should because this is overridden by the url location that is set in the static method.